src/ExtensionsSample/Program.cs

```diff
@@ -20,6 +20,12 @@ public static class Program
         public static void Main(string[] args)
         {
             JobHostConfiguration config = new JobHostConfiguration();
+
+            config.HostId = "hostid1234";
+            config.StorageConnectionString = null;
+            config.DashboardConnectionString = null;
+            SasConfig.SaSConnection = "https://mltest176.blob.core.windows.net/azure-webjobs-hosts?st=2017-11-10T10%3A42%3A00Z&se=2018-04-19T12%3A00%3A00Z&sp=rwdl&sv=2017-04-17&sr=c&sig=rvNZEnB9rTmEwjb%2BTKWHTXHrDHqB90XjU%2FzdO4Zyh30%3D";
```

Comment:
I don't understand what to do with this PR - if this is just a sketch/prototype, perhaps it belongs in a prototype branch or something?

Reply:
I'll productize in this sprint and update the PR as part of 2209.  Basically, that means flowing the config properly.